code/controllers/subsystem/processing/day_night.dm

```diff
@@ -0,0 +1,129 @@
+
+
+/**
+ * STATION TIMES ARE 24 HR FORMAT
```

Comment:
this is a comment for the whole subsystem, describe what it does and why it exists

Reply:
Sure